tools/devops/automation/scripts/GitHub.psm1

```diff
@@ -560,6 +560,22 @@ function New-GitHubSummaryComment {
         Write-Host "Artifacts were not provided."
     }
 
+    $githubPagePrefix = "https://xamarin.github.io/macios.ci"
+    if ([string]::IsNullOrEmpty($Env:PR_ID)) {
+        $githubPagePrefix = "$githubPagePrefix/ci/$Env:BUILD_SOURCEBRANCHNAME/$Env:BUILD_REVISION"
+    } else {
+        $githubPagePrefix = "$githubPagePrefix/pr/PR$Env:PR_ID"
+    }
+
+    $githubPagePrefix = "$githubPagePrefix/$Env:BUILD_BUILDID"
+    $sb.AppendLine("# GitHub pages")
+    $sb.AppendLine()
+    $sb.AppendLine("Results can be found in the following github pages (it might take some time to publish):")
+    $sb.AppendLine()
+    $sb.AppendLine("* [Test results]($githubPagePrefix/HtmlReport-sim/tests/vsdrops_index.html)")
+    $sb.AppendLine("* [API diff ]($githubPagePrefix/HtmlReport-sim/api-diff/api-diff.html)")
+    $sb.AppendLine("* [API & Generator diff]($githubPagePrefix/apicomparison/api-diff.html)")
```

Comment:
Suggestion: run a script with `$githubPagePrefix` as an argument

That way other people can change this without YAML (You Ask ManueL) and breaking things. Also makes it a lot easier to tests changes.

Even more important if (and I'm hoping for it) we move to have a pre-determined directory sync'ed up in GH pages. That would make it _much_ easier to add new things, like net6-based API diff or net6 `make compare` results, but just copying the files (to that pre-determined directory) and editing the "index" script to link to it.

Reply:
yep, I have to clean that a little